### ISSUE TITLE
Ebay

### ISSUE THREAD

**Issue by miktab** [mehboobafridi/mehboob-testing#1]
your bio says you're an amazon and ebay expert. 

I'm really interested in ebay api also. If you're still active, please reply to this and I would love to have a chat. I'm from Australia and I think we can work together to start a business.

**Comment by mehboobafridi** [comment]
Yes I am still active and working on eBay and Amazon APIs. Let me know as how may I help you.
Thanks

**Comment by miktab** [comment]
Awesome! Do you have a discord account or something we can communicate on?
my discord: partyhat#3429

**Comment by mehboobafridi** [comment]
We may use whatsapp if its easy for you. I haven't used discord before.

**Comment by miktab** [comment]
Can I have your email? I'd like to not post in public :)

**Comment by mehboobafridi** [comment]
mehboobafridi@yahoo.com
